dashboard/app/helpers/ai_system_prompts/system_prompt_helper.rb

```diff
@@ -1,46 +1,25 @@
 require 'cdo/shared_constants'
 
 # Helper methods for generating system prompts for AI shared across
-# AI Tutor and Evaluate Student Learning.
+# AI Tutor v1 and Evaluate Student Learning.
```

Comment:
@breville Just double checking that AI Tutor 2 doesn't also use this helper, right?